app/scripts/views/AppView.js

```diff
@@ -317,7 +321,7 @@ define([  'backbone',
 
     zoomresetClick: function(){
       if ( this.lookingAtViewer ){
-        zoomToFit();
+        this.model.trigger('zoomToFit');
```

Comment:
These should be method invocations on `ThreeViewer`.  See - `AppView` owns `ThreeViewer`, so there's no reason for it not to invoke these methods on it.  It's not exposing implementation details of `ThreeViewer` by doing so.  Further, there's no reason to trigger events if you don't have to.  Events have the tendency to trickle out.